drivers/include/periph/rtc.h

```diff
@@ -131,6 +131,22 @@ void rtc_poweroff(void);
  */
 void rtc_tm_normalize(struct tm *time);
 
+/**
+ * @brief Compare two time structs.
+ *
+ * @note  The time structs @p a and @p b are assumed to be normalized.
+ *        Use @see rtc_tm_normalize to normalize a struct tm that has been
```

Comment:
`@see` cannot be used in the middle of a sentence, according to [its documentation](http://www.doxygen.nl/manual/commands.html#cmdsa). You should use `@ref` here.

`@pre` is the proper way to [describe preconditions](http://www.doxygen.nl/manual/commands.html#cmdpre). If `@pre` is used consistently throughout RIOT, all preconditions will be displayed in the same way.